### PR TITLE
Ability to force AB Switch state for Cypress Tests

### PR DESCRIPTION
**Pull request by coldlink**
AB test switches are defined in Frontend and required to be "ON" for Cypress E2E Sign In Gate Test to successfully run and pass. However, in practice it is useful to be able to switch an AB test on and off in Frontend PROD Admin UI without having to make any code changes to a different repo.

In order not to break CI builds in DCR if someone switches off a Sign In Gate AB test in the Frontend Admin tool, we've added the ability to define a state that a switch should be in DCR, and will only run within a Cypress environment.

In #1877 @vlbee tried to do a similar thing by skipping tests if the switch was set to off in Frontend PROD, but this caused non-deterministic behaviour due to tests running asynchronously.

This presents a simpler solution, as we can set the state of the test switch in `cypressSwitches` which will only be picked up by Cypress, meaning that tests will still run regardless of the state in PROD.

@gtrufitt it would be useful to get your thoughts in this too?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,20 @@ Frontend rendering framework for theguardian.com. It uses [React](https://reactj
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 <!-- Automatically created with yarn run createtoc and on push hook -->
 
-- [Dotcom Rendering](#dotcom-rendering)
```
Comment (coldlink) [inline README.md:9]: not sure why this file updated automatically?

Comment (coldlink) [reply README.md:9]: since i didn't make changes to it

File: src/web/experiments/cypress-switches.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Forced AB Test Switches for Cypress testing
+
+// AB test switches are defined in Frontend and required to be "ON" for Cypress E2E Tests to successfully run and pass.
```
Comment (vlbee) [inline src/web/experiments/cypress-switches.ts:3]: Thanks - inline comments more helpful than docs to quickly orient devs

**Review by vlbee — APPROVED** [review]
I prefer this solution so giving it a ✅   but see what @gtrufitt thinks!

**Review by gtrufitt — APPROVED** [review]
LGTM!

**Pull request merged**